examples/protocol_specification_ex/fipa.yaml

```diff
@@ -36,8 +36,8 @@ ct:Description: |
 reply:
   cfp: [propose, decline]
   propose: [propose, accept, accept_w_inform, decline]
-  accept: [decline, match_accept, match_accept_w_inform]
-  accept_w_inform: [decline, match_accept, match_accept_w_inform]
+  accept: [match_accept, match_accept_w_inform, decline]
```

Comment:
All list entries should be alphabetically sorted

Reply:
Done